### PR TITLE
Added tlg4072.tlg003.1st1K-grc1.xml

### PR DESCRIPTION
**Pull request by sonofmun**
Eutocius Commentarius in libros de planorum aequilibriis

### CODE REVIEW

File: data/tlg4072/tlg003/tlg4072.tlg003.1st1K-grc1.xml [hunk 1]
```diff
@@ -0,0 +1,957 @@
...
+<listBibl>
+ <biblStruct>
+ <monogr>
+  <title xml:lang="fre">Archimède</title>
+  <editor>
+	<persName>
+	 <name>Charles Mugler</name>
+	</persName>
+  </editor>
+  <author ref="urn:cts:greekLit:tlg4072">Eutocius</author>
```
Comment (planatheisa) [inline data/tlg4072/tlg003/tlg4072.tlg003.1st1K-grc1.xml]: Should have `xml:lang="lat"`.

File: data/tlg4072/tlg003/tlg4072.tlg003.1st1K-grc1.xml [hunk 2]
```diff
@@ -0,0 +1,957 @@
...
+  <imprint>
+   <publisher xml:lang="fre">Les Belles Lettres</publisher>
+	<pubPlace>Paris</pubPlace>
+	<date>1972</date>
+  </imprint>
+  <biblScope unit="volume">4</biblScope>
+ </monogr>
```
Comment (planatheisa) [inline data/tlg4072/tlg003/tlg4072.tlg003.1st1K-grc1.xml]: Scan missing.

File: data/tlg4072/tlg003/tlg4072.tlg003.1st1K-grc1.xml [hunk 3]
```diff
@@ -0,0 +1,957 @@
...
+ταῦτα ἐν τῷ δευτέρῳ βιβλίῳ τῶν Ἀπολλωνίου Κωνικῶν.
+Ὅμως δὲ καὶ ἐπὶ τοῦ ΑΒΓ σχήματος καὶ ἐπὶ τοῦ &#9651;ΕΖ
+<lb n="10"/> τὸ κέντρον τοῦ βάρους, ἀφ᾿ οὗ δηλονότι ἀρτώμενον
+τὸ σχῆμα παράλληλόν ἐστι τῷ ὁρίζοντι, ἐντός ἐστι τῆς
+περιμέτρου· εἰ γὰρ ἔσται ἐπὶ τῆς περιμέτρου ἢ ἐκτός,
+ῥέψει ἐπὶ θάτερα· ὅπερ οὐχ ὑπόκειται.</p>
+</div>
+
+<div type="textpart" subtype="section" n="2">
+<head>Εἰς τὸ δ΄.</head>
```
Comment (planatheisa) [inline data/tlg4072/tlg003/tlg4072.tlg003.1st1K-grc1.xml]: Would it make sense to keep the greek numbering system here? It seems to be referring to something.

Comment (sonofmun) [reply data/tlg4072/tlg003/tlg4072.tlg003.1st1K-grc1.xml]: Good catch. It is a commentary on tlg0552.tlg005, so I will add the URNs in a corresp attribute, as discussed in #201

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `coverage`         |    -0.01 |
| `metadata_passing` |     2    |
| `metadata_total`   |     2    |
| `nodes_count`      |    19    |
| `texts_passing`    |    -1    |
| `texts_total`      |     1    |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |    +7282 |

## Units

| Changed                                                   | Status   |
|:----------------------------------------------------------|:---------|
| `./data/tlg4072/__cts__.xml`                              | New      |
| `./data/tlg4072/tlg003/__cts__.xml`                       | New      |
| `./data/tlg4072/tlg003/tlg4072.tlg003.1st1K-grc1.xml`     | New      |
| `./data/stoa0146d/stoa001/stoa0146d.stoa001.opp-grc1.xml` | Failing  |
| `./data/tlg0011/tlg003/tlg0011.tlg003.1st1K-grc1.xml`     | Failing  |
| `./data/tlg4072/__cts__.xml`                              | Passing  |
| `./data/tlg4072/tlg003/__cts__.xml`                       | Passing  |
| `./data/tlg4072/tlg003/tlg4072.tlg003.1st1K-grc1.xml`     | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3667)*

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +2 |
| `metadata_total`   |       +2 |
| `nodes_count`      |      +19 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |    +7282 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg4072/__cts__.xml`                          | New      |
| `./data/tlg4072/tlg003/__cts__.xml`                   | New      |
| `./data/tlg4072/tlg003/tlg4072.tlg003.1st1K-grc1.xml` | New      |
| `./data/tlg4072/__cts__.xml`                          | Passing  |
| `./data/tlg4072/tlg003/__cts__.xml`                   | Passing  |
| `./data/tlg4072/tlg003/tlg4072.tlg003.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3686)*

**Pull request merged**